go.mod

```diff
@@ -380,8 +384,8 @@ replace (
 	// https://github.com/osmosis-labs/wasmd/releases/tag/v0.45.0-osmo
 	github.com/CosmWasm/wasmd => github.com/osmosis-labs/wasmd v0.45.0-osmo
 
-	// cometbft is replaced to print custom app hash logs
-	github.com/cometbft/cometbft => github.com/osmosis-labs/cometbft v0.37.2-v21-osmo-1
+	// cometbft is replaced to print custom app hash logs. Using branch nicolas/v0.37.4-v23-osmo-1. Need to tag this before releasing
+	github.com/cometbft/cometbft => github.com/osmosis-labs/cometbft v0.0.0-20240201155058-95b5f9262e1d
```

Comment:
Update

Reply:
all for the comment change, but this has already been merged